telegram-base.c

```diff
@@ -402,14 +402,12 @@ gchar *get_download_dir (struct tgl_state *TLS) {
   if (dir) {
     g_free (dir);
   }
-#ifndef BITLBEE
+  if (g_strcmp0(purple_core_get_ui(), "BitlBee") == 0) {
+  dir = g_strconcat ("/tmp",  G_DIR_SEPARATOR_S, "downloads", NULL);
```

Comment:
Thanks!  I'm a bit unhappy about the missing indent, but the logic sounds good, and if it works for you and the "pidgin" path didn't change that's all I need.